keras_nlp/models/gpt_neox/gpt_neox_attention.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.models.gpt_neox.rotary_embedding import RotaryEmbedding
+from keras_nlp.utils.keras_utils import clone_initializer
+
```

Comment:
make sure to format your code with `./shell/format.sh` from the base of the repo. some of this does not look formatted correctly

Reply:
Hi @mattdangerw my black seems to be broken

Reply:
please install the latest version `23.3.0` (note that keras.io uses a pinned older version, but everywhere else we use the latest black)

Reply:
Now code format check in CI is Green. Thanks !